ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -178,11 +178,22 @@ def _vrf_default_ipv4_subnets(self) -> list[str]:
         return subnets
 
     @cached_property
-    def _vrf_default_ipv4_static_routes(self) -> list[str]:
+    def _vrf_default_ipv4_static_routes(self) -> tuple(list[str], bool, bool):
         """
-        Return list of ipv4 static routes in VRF "default"
+        Return tuple with:
+        [0]: list of ipv4 static routes in VRF "default"
+        [1]: boolean whether to redistribute static into the underlay protocol
+        [2]: boolean whether to redistribute static into overlay protocol for vrf default
+
+        For underlay, the boolean is set in the special cases where static routes should be redistributed
+        into the underlay, like when not running EVPN or not being a VTEP.
+
+        For overlay, the boolean is set if there are any static routes and this is an EVPN VTEP
+
+        The both booleans will respect "redistribute_static" under the VRF definition (default=True)
```

Comment:
For the sake of readability I am wondering if a dictionary would not be better than a tuple as a return value (rather than having `self._vrf_default_ipv4_static_routes[1]` in files)

Reply:
probably

Reply:
updated